distribution/docs/src/main/resources/_contents/_running/ingesting-contents.adoc

```diff
@@ -1,25 +1,18 @@
-==== Ingesting Data
+=== Ingesting Data
 
 Ingesting is the process of getting metadata into the Catalog Framework.
```

Comment:
We should up this to `metacard` instead of `metadata` since `metadata` is an attribute of a metacard